configuration/config-txt/pi4-hdmi.md

```diff
@@ -0,0 +1,24 @@
+# Raspberry Pi 4 HDMI pipeline
+
+In order to support dual displays, and modes up to 4k60, the Raspberry Pi 4 has updated the HDMI composition pipeline hardware in a number of ways. One of the major changes is that it generates 2 output pixels for every clock cycle.
+
+Every HDMI mode has a list of timings that control all the parameters around sync pulse durations. These are typically defined via a pixel clock, and then a number of active pixels, a front porch, sync pulse, and back porch for each of the horizontal and vertical directions. 
+
+Running everything at 2 pixels per clock means that the Pi4 can not support a timing where any of the horizontal timings are not divisible by 2. The firmware and Linux kernel will filter out any mode that does not fulfill this criteria.
+
+There is only one mode in the CEA and DMT standard that falls into this category - DMT mode 81, which is 1366x768 @ 60Hz. 1366x768 is already a unusual mode as normally the width of modes is divisible by 8, where 1366 is not.
```

Comment:
https://github.com/raspberrypi/documentation/blob/master/configuration/config-txt/video.md also lists DMT mode 86 as 1366x768 @ 60Hz ?

Reply:
How to explain it to the lay-person? Particularly when the parts of the spec you wish to reference aren't public docs.

I'd referenced front and back porches and sync widths.
Someone (I'm guessing Dom) has posted the full timing table we use at https://pastebin.com/vVLVTMin. It's a direct lift from our firmware source tree.
mode 81
```
{ "1366x768@60", { 81,  1366, HDMI_POSITIVE_SYNC,   70, 143, 213,  768,  HDMI_POSITIVE_SYNC,  3,  3,  24,    0,    0,       0,     60,      HDMI_NONINTERLACED,  85500000},        HDMI_ASPECT_16_9},
```
The 143 and 213 are the problems - they aren't even numbers.
Mode 86 is 1366x768 "reduced blanking"
```
 { "1366x768rb",  { 86,  1366, HDMI_POSITIVE_SYNC,   14,  56,  64,  768,  HDMI_POSITIVE_SYNC,  1,  3,  28,    0,    0,       0,     60,      HDMI_NONINTERLACED,  72000000},        HDMI_ASPECT_16_9},
```
No odd numbers in the horizontal timings, so all good.

The kernel does have this same information at https://github.com/raspberrypi/linux/blob/rpi-5.4.y/drivers/gpu/drm/drm_edid.c#L391 but they use cumulative numbers, so
```
	{ DRM_MODE("1366x768", DRM_MODE_TYPE_DRIVER, 85500, 1366, 1436,
		   1579, 1792, 0, 768, 771, 774, 798, 0,
		   DRM_MODE_FLAG_PHSYNC | DRM_MODE_FLAG_PVSYNC) },
```
The troublesome numbers are still there but less obviously:
1579-1436 = 143
1792-1579 = 213

Feel free to reword, but it's a struggle without getting into huge amounts of technical detail that is likely to confuse or bemuse many.
Possibly just reference that it is the sync width and back porch that are odd values?

The kernel headers do include a pretty ASCII diagram of what the sync periods mean if you want to reference that - https://github.com/raspberrypi/linux/blob/rpi-5.4.y/include/drm/drm_modes.h#L198